test/CMakeLists.txt

```diff
@@ -30,13 +27,13 @@ foreach( TARGET_ALIAS template_debug template_release editor )
             src/tests.h
     )
 
-    set( OUTPUT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/project/bin/" )
-
     # conditionally add doc data to compile output
-    if( TARGET_ALIAS MATCHES "editor|template_debug" )
-        target_sources( ${TARGET_NAME} PRIVATE "${DOC_DATA_SOURCE}" )
+    if( TARGET_ALIAS MATCHES "editor" )
+        target_doc_sources( ${TARGET_NAME} ${DOC_XML} )
     endif()
```

Comment:
Why does this remove "template_debug"? We do want the docs included for "template_debug" as well